streams/src/main/java/org/apache/kafka/streams/StreamsBuilder.java

```diff
@@ -66,207 +68,74 @@
      * If this is not the case it is the user's responsibility to repartition the data before any key based operation
      * (like aggregation or join) is applied to the returned {@link KStream}.
      *
-     * @param topics the topic names; must contain at least one topic name
+     * @param topic the topic names; must contain at least one topic name
```

Comment:
nit: `topic name` (no plural) -- remove `must contain...` -- add `cannot be {@code null}`